test/test_tagging.py

```diff
@@ -0,0 +1,110 @@
+import json
+from unittest.mock import (
+    PropertyMock,
+    patch,
+)
+
+from azul.deployment import (
+    populate_tags,
+)
+from azul.logging import (
+    configure_test_logging,
+)
+from azul_test_case import (
+    AzulUnitTestCase,
+)
+
+
+def setupModule():
+    configure_test_logging()
+
+
+class TestTagging(AzulUnitTestCase):
```

Comment:
```suggestion
class TestTerraformResourceTags(AzulUnitTestCase):
```

and name module accordingly.